src/app/routes/onDemandTV/getInitialData/index.js

```diff
@@ -1,11 +1,25 @@
 import path from 'ramda/src/path';
 import fetchPageData from '../../utils/fetchPageData';
 import overrideRendererOnTest from '../../utils/overrideRendererOnTest';
+import getPlaceholderImageUrlUtil from '../../utils/getPlaceholderImageUrl';
```

Comment:
```suggestion
import getPlaceholderImageUrl from '../../utils/getPlaceholderImageUrl';
```
do we need to have `util` at the end of the import name?

Reply:
Will remove!